app/assets/stylesheets/components/form.scss

```diff
@@ -6,3 +6,35 @@ input.btn.btn-default.btn-success.btn-block {
   font-weight: bold;
   font-size: $base-font-size;
 }
+
+.form-section {
+	background-color: $white;
```

Comment:
Damn it. Does anyone know how to force Sublime into making this default. Every time I open a file, I have to manually change these settings.